src/uct/api/v2/uct_v2.h

```diff
@@ -439,6 +440,18 @@ typedef struct uct_md_mem_reg_params {
      * dmabuf region, then this field must be omitted or set to 0.
      */
     size_t                       dmabuf_offset;
+    
+    /**
+     * Device Memory
+     * 
+     * If not set (along with its corresponding bit in the field_mask -
+     * @ref UCT_MD_MEM_REG_FIELD_DEVICE_MEMORY) it's assumed to be NULL.
+     */
+#if HAVE_IBV_DM
+    struct ibv_dm                *dm;
+#else
+    void                         *dm;
+#endif
```

Comment:
Will use `uint32_t dm_handle` instead

Reply:
why it has to be in the API?

Reply:
@roiedanino as we discussed, no need to put it in the API at all.

Reply:
When registering a dm we will need to refer to it, I'm planning to change the implementation to use `ibv_import_dm` with the handle instead of an `ibv_dm` pointer

Reply:
`ibv_reg_dm_mr` is going to need a pointer to the `dm`